src/puya/resolve-puya-path.ts

```diff
@@ -9,6 +9,7 @@ export async function resolvePuyaPath(): Promise<string> {
   const version = parseSemVer(Constants.targetedPuyaVersion)
 
   const puyaStorageDir = getPuyaStorageDir()
+  // return {'path to service mode version of puya'} # TODO: remove once binary on npm contains --service mode
```

Comment:
What would need to change once the binary contains service mode? Wouldn't this code stay the same?